website/src/pages/docs/recipes/webpack.mdx

```diff
@@ -50,18 +51,17 @@ class ProfileComponent {
 ```
 
 
-### Optional: Install and configure a custom Webpack configuration (when using Angular CLI)
+## Optional: Install and configure a custom Webpack configuration (when using Angular CLI)
 
 Install `@angular-builders/custom-webpack` with the following command:
 
-```
-npm i -d @angular-builders/custom-webpack
-```
+<PackageCmd packages={['@angular-builders/custom-webpack']} />
```

Comment:
Should be object with isDev: true

Reply:
<PackageCmd packages={['@angular-builders/custom-webpack']} isDev={true} />

@B2o5T  like that?